internal/hcs/system.go

```diff
@@ -407,6 +407,33 @@ func (computeSystem *System) Resume(ctx context.Context) (err error) {
 	return nil
 }
 
+// Save the compute system
+func (computeSystem *System) Save(ctx context.Context, options string) (err error) {
```

Comment:
Is there a defined schema type for what `options` can be? Would it be better to make this take a `options interface{}` and do the marshalling in this function, so it matches the behavior of `CreateComputeSystem`?

Reply:
Ping on this

Reply:
We do have a schema for SaveOptions. I have updated the code to use that.